bndtools.builder/src/org/bndtools/builder/classpath/BndContainerInitializer.java

```diff
@@ -210,17 +214,31 @@ public static void setClasspathEntries(IJavaProject javaProject, IClasspathEntry
         }
 
         ArrayList<IClasspathEntry> result = new ArrayList<IClasspathEntry>(containers.size());
-        LinkedHashMap<Project,List<IAccessRule>> projectAccessRules = new LinkedHashMap<Project,List<IAccessRule>>();
+        LinkedHashMap<Project,List<IAccessRule>> projectAccessRulesExports = new LinkedHashMap<Project,List<IAccessRule>>();
+        LinkedHashMap<Project,List<IAccessRule>> projectAccessRulesPrivates = new LinkedHashMap<Project,List<IAccessRule>>();
         for (Container c : containers) {
             if (c.getType() == TYPE.PROJECT && c.getError() == null) {
-                calculateWorkspaceBundleAccessRules(projectAccessRules, c);
+                calculateWorkspaceBundleAccessRules(projectAccessRulesExports, projectAccessRulesPrivates, c, model);
+            }
+        }
+        //[cs] Add ** NO access to end of rule sets that have rules.
```

Comment:
Habit of mine. It's a quick way of knowing who added the comment. I find it useful. If the bndtools coding standard says don't do that, I can remove it.